InvenTree/config_template.yaml

```diff
@@ -158,7 +158,7 @@ plugins_enabled: False
 
 # Set this variable to True to enable auto-migrations
 # Alternatively, use the environment variable INVENTREE_AUTO_UPDATE
-auto_update: False
+auto_update: True
```

Comment:
Is this intentionally changed?

Reply:
@matmair nope, meant to change my local setup file